config_converter/config_mapper/config_mapper.py

```diff
@@ -30,15 +30,51 @@
 _SUPPORTED_PLUGINS = ['in_tail']
 
 
+def initialize_stats(d: config_pb2.Directive) -> dict:
+    """Initializes the stats dict to print out."""
+    stats = {
+        'attributes_num': get_num_attrs(d),
+        'attributes_recognized': 0,
+        'attributes_unrecognized': 0,
+        'attributes_skipped': 0,
+        'entities_num': get_num_ents(d),
+        'entities_skipped': 0,
+        'entities_unrecognized': 0,
+        'entities_recognized_success': 0,
+        'entities_recognized_partial': 0,
+        'entities_recognized_failure': 0
+    }
+    return stats
+
+
+def get_num_attrs(d: config_pb2.Directive) -> int:
+    """Takes a directive, returns total no of attributes."""
```

Comment:
```suggestion
    """Takes a directive, returns total number of attributes rooted from this directive."""
```

I think this function returns not the number of its own attribute, hence the suggestion.

Reply:
Will rewrite the comment to make it more clear!